docs/release_notes.rst

```diff
@@ -0,0 +1,7 @@
+=============
+Release notes
+=============
+
+.. towncrier release notes start
```

Comment:
Ah, interesting, do you also want to setup Towncrier?

Reply:
wasnt intending to in this PR, maybe later.

this was a copy paste error... sorry